docs/customization/route_config.md

```diff
@@ -12,3 +16,49 @@ Then add the routes to your customized controllers:
 $routes->get('login', '\App\Controllers\Auth\LoginController::loginView');
 $routes->get('register', '\App\Controllers\Auth\RegisterController::registerView');
 ```
+
+After customization, check your routes with the [spark routes](https://codeigniter.com/user_guide/incoming/routing.html#spark-routes) command.
+
+## Use Locale Routes
+
+You can use the `{locale}` placeholder in your routes
```

Comment:
And the way to use the following items does not change?

https://github.com/codeigniter4/shield/blob/112ba2a15734b59255186eb7f65f8f727a3885f6/src/Config/Auth.php#L67

Reply:
`login` is a named route, so not change.